packages/core/package.json

```diff
@@ -51,16 +51,20 @@
   "dependencies": {
     "@metriport/commonwell-sdk": "file:packages/commonwell-sdk",
     "@opensearch-project/opensearch": "^2.3.1",
+    "@playwright/browser-firefox": "^1.39.0",
```

Comment:
OOC - why not chrome?

Reply:
It worked for a few times, then it started failing. I moved to FF and it worked, and have been working ever since. TBH, I didn't want to invest too much time on this, its harder to debug.